heap_sort/HeapSort.java

```diff
@@ -0,0 +1,44 @@
+/*
+*  Following is the implementation of HeapSort algorithm
+*  It will sort the input array with time complexity O(N*logN)
+*  N being the size of array
+*/
+
+public class HeapSort {
+
+	public static void heapify(int[] arr, int size, int parent) {  // Build max-heap of subtree of parent index
```

Comment:
I am absolutely not doubting your eternal experience, :wink:  but I personally think it needs to be specified the right algorithm to build the right heap.

`heapify(arr, arr.length, i);  // Create max-heap`
or 
`max_heapify(arr, arr.length, i);  // Create max-heap`

Not good?